components/PlayerDetailsList/PlayerDetailsList.tsx

```diff
@@ -316,6 +317,23 @@ function PlayerDetailsList(props: Props) {
 
     return (
         <div className={styles.playerDetailsList}>
+            {!showFilter() ? (
+                <Tooltip
+                    content={
+                        <p>
+                            How to filter auctions/bids <HelpIcon style={{ color: '#007bff' }} />
+                        </p>
+                    }
+                    hoverPlacement="bottom"
+                    type="hover"
+                    tooltipContent={
+                        <>
+                            <p>Claim your account to filter your own auctions/bids.</p>
+                            <p>If you have starter premium or above you are able to use the filter for any player.</p>
+                        </>
+                    }
```

Comment:
![bad centered](https://user-images.githubusercontent.com/16632490/201374138-ba6f51b5-c05a-46a9-8bee-1e619b31dd20.png)
should be below the information icon not in the center.  
Maybe add a redirection possibility to /premium as well